server.js

```diff
@@ -455,6 +455,23 @@ var handlerWrapper = module.exports.handlerWrapper = function handlerWrapper(pin
       });
       return;
     }
+
+    if (message.indexOf('watch queue') > -1) {
+      if (!queueWatchers.includes(from)) {
+        queueWatchers.push(from);
+      }
+
+      bot.say(to, `i'll let you know when the queue is ready for you ${from}`);
+    }
+
+    if (message.indexOf('stop watching queue') > -1) {
+      const nickNameIndex = message.indexOf(from);
```

Comment:
Shouldn't this be `queueWatchers.indexOf`?

Reply:
Yes, thanks for reviewing. I've made the change to the code.